test/throughput/test_buffer_profile.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+from tianshou.data import (ListReplayBuffer, PrioritizedReplayBuffer,
+                           ReplayBuffer)
+
+
+@pytest.fixture(scope="module")
+def data():
+    np.random.seed(0)
+    obs = {'observable': np.random.rand(
+        100, 100), 'hidden': np.random.randint(1000, size=200)}
+    info = {'policy': "dqn", 'base': np.arange(10)}
+    add_data = {'obs': obs, 'rew': 1., 'act': np.random.rand(30),
+                'done': False, 'obs_next': obs, 'info': info}
+    buffer = ReplayBuffer(int(1e3), stack_num=100)
+    buffer2 = ReplayBuffer(int(1e4), stack_num=100)
+    Slice = slice(-3000, -1000, 2)
+    indexs = np.random.choice(int(1e3), size=3, replace=False)
+    return{
+        'add_data': add_data,
+        'buffer': buffer,
+        'buffer2': buffer2,
+        'Slice': Slice,
```

Comment:
Personally I prefer the following, because local variable names should not be lower-case.
```python
'slice': slice(-3000, -1000, 2)
```